walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -6,15 +6,15 @@
 
 = Running the mobile showcase application with {mobile-services-name}
```

Comment:
I am thinking maybe we should call it "GraphQL showcase application" rather than "mobile". Also I think we should remove `{mobile-services-name}` at the end.

@pwright wdyt?

Reply:
Awesome idea!